cmd/query/app/http_handler_test.go

```diff
@@ -303,12 +305,16 @@ func TestGetTrace(t *testing.T) {
 	for _, tc := range testCases {
 		testCase := tc // capture loop var
 		t.Run(testCase.suffix, func(t *testing.T) {
-			reporter := jaeger.NewInMemoryReporter()
-			jaegerTracer, jaegerCloser := jaeger.NewTracer("test", jaeger.NewConstSampler(true), reporter)
-			jTracer := jtracer.OT(jaegerTracer)
-			defer jaegerCloser.Close()
-
-			ts := initializeTestServer(HandlerOptions.Tracer(jTracer))
+			exporter := tracetest.NewInMemoryExporter()
+			tracerProvider := sdktrace.NewTracerProvider(
+				sdktrace.WithSyncer(exporter),
+				sdktrace.WithSampler(sdktrace.AlwaysSample()),
+			)
+			otTracer, wrappedTracerProvider := otbridge.NewTracerPair(tracerProvider.Tracer(""))
```

Comment:
Why don't we give the tracer a name here?

Reply:
@yurishkuro LMK if we need to give it any name here.

Reply:
I don't think so, this is a test and you're not going to validate the output of OT Bridge.

Are you sure you even need OpenTracing tracer for this test? What if keep it as nil? Might just work.

Reply:
That worked. Thanks